### PR TITLE
Vps 029/authentication and dashboard button

### PR DESCRIPTION
**Pull request by mlai962**
## Describe the issue

Create a page that’s only accessible to lecturers when they log in where they can see the dashboard (and other students can't access it when they log in)

## Describe the solution

Created a button that leads to an empty dashboard page. Tried to reuse the top bar that was already created but there were errors.

We are able to determine whether or not the current user is a lecturer, depending on the email domain. However, security is an issue as it is all done on the frontend.

## Risk
Slight security risk since authentication is currently in frontend but we will be moving it to the backend. 

## Definition of Done

- [ ] Code peer-reviewed
- [ ] Wiki Documentation is written and up to date
- [ ] Unit tests written and passing
- [ ] Integration tests written and passing
- [ ] Continuous Integration build passing
- [ ] Acceptance criteria met
- [ ] Deployed to production environment

### CODE REVIEW

**Comment by mlai962** [comment]
When I ran the tests for the first time there was only one error because of a few unused imports, however, after removing the imports there are now two errors and my VPS shows an error when I try to run it. This is my first time running the tests and I don't know how to fix the ones that are still there, if anyone could help it would be much appreciated.

**Comment by nickykay98** [comment]
@mlai962 I'd presume this may have to do with @lia-arroyo's work on the frontend - Lia, please this check when you can and let us know if it's all good to merge the authentication stuff you've done for now in this PR

**Comment by lia-arroyo** [comment]
Hi Matthew and Nicholas, I've got an assignment due tonight, so sadly I can't fix it right now, but I will have a look into it tomorrow night! 😊

File: frontend/src/firebase/StaffAuth.js [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import React, { useState } from "react";
```
Comment (lia-arroyo) [inline frontend/src/firebase/StaffAuth.js]: I think we only needed to remove the "React, " from the import, not the entire line

**Comment by lia-arroyo** [comment]
Hi @nickykay98 and @mlai962, I updated the snapshots, and the tests are passing now. 

It should be okay to merge into the main branch, but the dashboard button is still not authenticated and leads to an empty page. Should we wait until I actually get the lecturer authentication to work in the backend before merging?

**Comment by nickykay98** [comment]
@lia-arroyo Thanks! I think it should be fine to keep it unauthenticated while you keep working on it in the backend in this branch, since Matthew will be working with Alfred to implement the prototype dashboard on another branch and Aden will just be using dummy data to connect the backend with the frontend

**Pull request merged**